.github/workflows/discord-bot.yml

```diff
@@ -0,0 +1,23 @@
+name: Notify Discord on PR Merge
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
since we don't use a `main` branch for docs, you might have to figure out how to dynamically fetch the default branch for the docs repo...